src/main/resources/static/main-page.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Main Manu</title>
+    <meta name="description" content="Main Page">
+    <link rel="stylesheet" href="css/style2.css">
+</head>
+<body>
+    <div class='conatiner'>
+        <nav class='toppanel'>
+            <ul>
+                <li><button type="button">Main Page</button></li>
+                <li>Followed summoners</li>
+                <li><button type="button">Setting</button></li>
+                <li><button type="button">Account</button></li>
+                <li><button type="button">Log out</button></li>
+            </ul>
+        </nav>
+        <nav class='leftpanel'>
+            <ul>
+                <li>Summoner Name</li>
+                <li><img alt="summoner icon" src="lolicon.jpg"/></li>
+                <li><img alt="summoner division" src="DiamondBadge.png"/></li>
+                <li>Summoner Division Number</li>
+            </ul>
+        </nav>
+        <nav class='rightpanel'>
+            <ul>
+                <li>Summoner1</li>
```

Comment:
each should contain summoner name, icon, game type (ranked/normal), game time. Use fake data.